xclim/indices/_conversion.py

```diff
@@ -5,7 +5,7 @@
 
 import numpy as np
 import xarray as xr
-from numba import float32, float64, vectorize  # noqa
+from numba import float32, float64, njit, vectorize  # noqa
```

Comment:
```suggestion
from numba import float32, float64, vectorize  # noqa
```
I think this change is now superfluous ?

Reply:
Indeed !